pkgs/development/python-modules/snapcast/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, buildPythonPackage, fetchPypi, construct }:
+
+buildPythonPackage rec {
+  pname = "snapcast";
+  version = "2.0.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0hlfcg0qdfavjzhxz4v2spjkj6440a1254wrncfkfkyff6rv9x3s";
+  };
+
+  propagatedBuildInputs = [ construct ];
+
+  # I need to see why the tests are not running
+  doCheck = false;
```

Comment:
Did you try
```
checkPhase = ''
  py.test
'';
```
?

Reply:
No tests in tarball. https://github.com/happyleavesaoc/python-snapcast/issues/23